core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/TagsImpl.java

```diff
@@ -88,10 +91,19 @@
         final String[] tagIds = asset.getProperties().get(tagPropertyName, new String[]{});
```

Comment:
i know you didnt write this :) but this line is really weird.... cq:tags would be String[] but tagTitles would be List<String>... oof